Dockerfile

```diff
@@ -146,7 +146,12 @@ RUN for postgisv in ${POSTGIS_VERSIONS}; do \
 # This need to be done after the PostgreSQL packages have been installed,
 # to ensure we have the preferred libpq installations etc.
 RUN apt-get install -y python3-etcd python3-requests python3-pystache python3-kubernetes python3-pysyncobj
-RUN apt-get install -y patroni
+RUN apt-get install -y patroni=2.1.3-\*
+# Patch Patroni code with changes from https://github.com/timescale/patroni/pull/1
+# NOTE: This is a temporary solution until changes land upstream.
+RUN wget -qO- https://raw.githubusercontent.com/timescale/patroni/v2.2.0-beta.1/patroni/ha.py > /usr/lib/python3/dist-packages/patroni/ha.py && \
```

Comment:
I think we should make this part optional (perhaps using docker variables) and keep this disabled by default for now.

Reply:
That's a good idea. We can add the param to a build matrix so that we can stamp-out a version with these changes, and a version without. Then if we ever need to switch due to bugs &c, we can do so.

Reply:
Ok, I think I've got a working solution. Testing now.